frontend/src/styles/main.scss

```diff
@@ -1,4 +1,73 @@
-body, html {
-  background-color: deeppink;
+$background-color: #171717;
+$card-color: #2F3138;
+$table-color: #212121;
+$border-color: #737784;
+$contrast-color: #FFFFFF;
+$alert-color: #DC3545;
+$success-color: #28A745;
+
+html {
   height: 100%;
+  margin: 0;
+
+  body {
+    margin: 0;
+
+    .app {
+      display: grid;
+      grid-template-columns: repeat(5, 1fr);
+      grid-template-rows: repeat(5, 1fr);
+      grid-column-gap: 24px;
+      grid-row-gap: 24px;
+      padding: 40px;
+      background-color: $background-color;
+      color: $contrast-color;
+      font-family: Helvetica Neue, sans-serif;
+      font-size: 16px;
+      height: 100vh;
+      position: relative;
+      box-sizing: border-box;
+    }
+  }
+}
+
+h1 {
+  font-size: 2.5rem;
+  margin: 0;
+}
+
+h2 {
+  font-size: 2rem;
+  margin: 0;
+}
+
+h3 {
+  font-size: 1.5rem;
+  margin: 0;
+}
+
+p {
+  font-size: 1rem;
+  margin: 0;
+}
+
+button {
+  height: 50px;
+  width: 100%;
+  background: transparent;
+  border-radius: 8px;
+  border: 2px solid $contrast-color;
+  color: $contrast-color;
+  margin-top: 24px;
+  cursor: pointer;
+  font-size: 1.125rem;
+  transition: 0.3s;
+}
+
+.c-icon-button {
+  background: none;
+  border: none;
+  cursor: pointer;
+  padding: 0;
+  width: auto;
 }
```

Comment:
This should be extracted into an `IconButton` component along with the corresponding HTML.

Reply:
✅This is now implemented